### ISSUE TITLE
APInf.io as global catalog with harvesting

### ISSUE THREAD

**Issue by kyyberi** [apinf/platform#2113]
We want apinf.io contain a lot of APIs. At the moment it seems pretty hard to grow amount of APIs in the catalog with current approach. 

## Automated harvesting

One way to get amount of APIs in catalog rising is to build harvesting system and get all APIs from dedicated apinf installations to apinf.io automatically.  Dedicated apinf owners can block this feature in configuration. We have build a harvester which knows about all the installations. This means that each time new apinf is installed and taken into use, it will use API to notify harvester that I'm here. Harvester pulls information from apin installations to own database. Harvester is active participant here, not the standalone apinf installations. From there it periodically pushes changed information to apinf.io via API. 

![harvester](https://cloud.githubusercontent.com/assets/1927158/22930778/a48c7720-f2ca-11e6-8af0-382f9e9d2bbb.png)

## Crowd sourced in parallel
In addition, all our employees and community members should be encouraged to add APIs to apinf.io directly. I did that with apisuomi.fi and no one complained. 

## Task
Research the subject and draft a plan how to proceed.

**Comment by kyyberi** [comment]
Related to #2114

**Comment by philippeluickx** [comment]
I would argue this is out of scope for apinf.io. Seperate project is available on github and we can integrate through apinf.io REST API.

**Comment by ilarimikkonen** [comment]
nope
